RULES.md

```diff
@@ -695,6 +695,116 @@ Unimplemented rules from the AWS PCI DSS 3.2.1 Conformance Pack.
 | [ssm-document-not-public](https://docs.aws.amazon.com/config/latest/developerguide/ssm-document-not-public.html)                                                                 | SSM documents are not public.                                                                                                       | 1.2, 1.2.1, 1.3, 1.3.1, 1.3.2, 1.3.4, 1.3.6, 2.2.2                                                                |
 | [vpc-sg-open-only-to-authorized-ports](https://docs.aws.amazon.com/config/latest/developerguide/vpc-sg-open-only-to-authorized-ports.html)                                       | The VPC Security Group restricts IPv4 TCP traffic on unauthorized ports.<sup>[2](#vpc-sg-open-only-to-authorized-ports)</sup>       | 1.2, 1.2.1, 1.3, 1.3.1, 1.3.2, 2.2.2                                                                              |
 
+## NZISM 3.6
+
+The Operational Best Practices for NZISM3.6 provides a sample mapping between the New Zealand Government Communications Security Bureau (GCSB) Information Security Manual (NZISM) and AWS Managed Config rules. Each Config rule applies to a specific AWS resource, and relates to one or more NZISM controls. An NZISM control can be related to multiple Config rules
+
+### Errors
```

Comment:
These should reflect the changes suggested below, additionally we should call out which controls are `SHOULD` and `MUST`